packages/ifpack2/test/unit_tests/Ifpack2_UnitTestOverlappingRowMatrix.cpp

```diff
@@ -416,9 +556,82 @@ TEUCHOS_UNIT_TEST_TEMPLATE_3_DECL(Ifpack2OverlappingRowMatrix, getLocalDiag, Sca
     TEST_EQUALITY(ldGids[i],ovrmGids[i]);
 }
 
+
+TEUCHOS_UNIT_TEST_TEMPLATE_3_DECL(Ifpack2OverlappingRowMatrix, reducedMatvec, Scalar, LocalOrdinal, GlobalOrdinal)
```

Comment:
It should be perfectly harmless to assume default `LocalOrdinal` and `GlobalOrdinal` here.